modules/model/shared/src/main/scala/gem/geom/GmosScienceAreaGeometry.scala

```diff
@@ -0,0 +1,119 @@
+// Copyright (c) 2016-2020 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package gem.geom
+
+import gem.`enum`.{GmosNorthFpu, GmosSouthFpu}
+import gsp.math.{Angle, Offset}
+import gsp.math.geom._
+import gsp.math.geom.syntax.all._
+import gsp.math.syntax.int._
+
+/**
+ * GMOS science area geometry.
+ */
+object GmosScienceAreaGeometry {
+
+  val imaging: ShapeExpression =
+    imagingFov(330340.mas, 33840.mas)
+
+  val mos: ShapeExpression =
+    imagingFov(314240.mas, 17750.mas)
+
+  def shapeAt(
+    posAngle:    Angle,
+    offsetPos:   Offset,
+    fpu:         Option[Either[GmosNorthFpu, GmosSouthFpu]]
+  ): ShapeExpression =
+    shapeFromFpu(fpu) ↗ offsetPos ⟲ posAngle
+
+  private def shapeFromFpu(fpu: Option[Either[GmosNorthFpu, GmosSouthFpu]]): ShapeExpression =
```

Comment:
Maybe there is a less dumb way to do this?

Reply:
In which case would it be None?

Reply:
For imaging.  The shape is defined but for XT we'll only really be supporting longslit.